R/learner_glmnet_surv_cv_glmnet.R

```diff
@@ -53,15 +65,16 @@ delayedAssign(
           mxitnr               = p_int(1L, default = 25L, tags = "train"),
           nfolds               = p_int(3L, default = 10L, tags = "train"),
           nlambda              = p_int(1L, default = 100L, tags = "train"),
-          offset               = p_uty(default = NULL, tags = "train"),
+          offset               = p_uty(default = NULL, tags = c("train", "predict")),
+          newoffset            = p_uty(tags = "predict"),
```

Comment:
I don't understand this, why does `offset` now have two tags but there is also a new `newoffset` parameter?

Reply:
`offset` is needed in the `.predict` method and it says so also in the [documentation](https://www.rdocumentation.org/packages/glmnet/versions/4.1/topics/survfit.coxnet)

`offset` is a vector the size of the train set and if used during training, it is also needed during prediction along with a `newoffset` (vector the size of the test set).